docs/Creator-V2-Whats-New.md

```diff
@@ -1,309 +1,195 @@
 
+Most of the UI elements in the new Survey Creator (Property Grid, Translation and Logic Tabs, modal editors) are stylized surveys from our own SurveyJS Library for React. This feature allows you to integrate any 3rd-party React component into both Survey Creator and SurveyJS library. View examples that show how to integrate the [React Select](https://react-select.com/home) component [into a standalone survey](https://surveyjs.io/Examples/Library?id=custom-widget-react-select) or [into the Survey Creator](https://surveyjs.io/Examples/Survey-Creator?id=react-select).
 
 <a id="ui-changes"></a>
 
-## Major changes in UX/UI
-We have introduced a lot of changes in Creator V2. Actually, we have rewritten almost all code from scratch to make Creator's UI cleaner, more intuitive and adaptive.
-
-
-<a id="page-navigation"></a>
-
-### Page navigation improved
-We have completely changed the way a user works with surveys that have multiple pages. Instead of displaying pages in horizontally oriented tabs, we now use a Microsoft Word inspired page layout (all pages, one by one) with a vertically oriented page navigator widget.
+## Major UI/UX changes
 
-![Page navigator](images/creator-v2-page-navigator.png)  
-_Page navigator_
+We redesigned most parts of the Survey Creator to make its UI more adaptive and UX more intuitive. Refer to help topics in this section for details.
 
-<!-- 
-<p align="center">
-  
-![Page navigator](images/creator-v2-page-navigator.png)
-
-_Page navigator_
-</p> 
--->
+<a id="toolbox-and-property-grid"></a>
 
+### New Toolbox and Property Grid design
 
+In the new design, the Toolbox and Property Grid look modern and adapt themselves to different layouts.
 
+<img src="./images/creator-v2-toolbox-and-property-grid.png" alt="Survey Creator - Toolbox and Property Grid" width="75%">
 
 <a id="action-buttons"></a> 
 
-### Action buttons rearranged
+### Action buttons relocated
 
-Action buttons used to be split between different tabs (Designer, Logic, etc.). Now, these actions are displayed within the question that you configure and can also be accessed programmatically.
+Action buttons used to be displayed above the question you configure. Now, these buttons are displayed within the question and can be accessed programmatically.
```

Comment:
Passive...

```suggestion
We now display action buttons within their corresponding question (instead of above the question). Programmatic access to the buttons is now possible.
```